app/src/androidTest/java/com/sdp/movemeet/view/activity/ActivityNotNullDescriptionTest.java

```diff
@@ -41,7 +42,7 @@
     private final static double DUMMY_DURATION = 10.4;
     private final static Sport DUMMY_SPORT = Sport.Running;
     private final static String DUMMY_ADDRESS = "address";
-    public FirebaseAuth mAuth;
+    public FirebaseAuth fAuth;
```

Comment:
Nice rename !

Reply:
Yes! This is again to follow our agreed convention!